tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -146,7 +167,7 @@
     SpanId: Id_9,
     Name: OtherLibrary.internal,
     Resource: Response,
-    Service: OtherLibrary,
+    Service: MyServiceName,
```

Comment:
This is intentional and called out in the PR description. And since this is no longer a top-level span, the corresponding metric is removed